travvy/src/components/AttractionDetails.vue

```diff
@@ -3,24 +3,31 @@
     <!-- Button to go to go back to the AttractionsList page --> 
     <router-link to="/AttractionList" tag="button" class="back"><span>Back</span></router-link>
 
-    <h1>CN Tower </h1>
+    <h1>{{$store.state.recommendedAttractions[0].attraction_name}} </h1>
     <img src="../images/toronto.jpg" alt="Picture of Toronto"/>
     <br>
     <h1>Description</h1>
     <p>The CN Tower is a 553.3 m-high concrete communications and observation tower located in Downtown Toronto, Ontario, Canada.</p>
-    <h3>Address:</h3>
-    <h3>Email:</h3>
-    <h3>Phone Number:</h3>
-    <h3>Ticket Pricing:</h3>
-    <h3>Maximum Group Size:</h3>
-    <h3>Are masks required:</h3>
-    <h3>COVID-19 Safety Measures:</h3>
+    <h2>Address: <span>{{$store.state.recommendedAttractions[0].street_address}}</span> </h2>
```

Comment:
this is great! Thanks for making it dynamic.

Reply:
the only thing I am concerned about is the [0] index. Wouldn't that only show the first attraction for each city? I did some unit tests and for both Toronto and London only the first attraction would show up regardless of which one I picked. This is something we can fix on the next sprint.

Reply:
you are correct, i was running into trouble with integrating this feature to be fully dynamic for each attraction however I was able to figure out how to make the first attraction for each city dynamically display, for example if you swapped between toronto and london, it would change the attraction details for the cities to te first available attraction from tr DB